node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+
+for (var x = 2; x < process.argv.length; x ++) {
+  fs.readFile(process.argv[x], 'utf8', (err, data) => {
+    console.log(data);
+  });
+}
```

Comment:
It only needed to read one file, but great job making it read unlimited files 👍

Reply:
ya i assumed that was going to be something later so i just did that when making it